utils/build-presets.ini

```diff
@@ -2689,6 +2689,8 @@ swift-freestanding-triple-name=macosx11.0
 swift-freestanding-module-name=macos
 swift-freestanding-archs=x86_64
 build-swift-private-stdlib=0
+swift-stdlib-use-relative-protocol-witness-tables=1
```

Comment:
Can this be instead done in the parent of all the "minimal footprint" settings, i.e. in `mixin_stdlib_minimal`?

Reply:
mixin_stdlib_minimal might be referenced by configurations that we don't intended to build with that feature enabled

Reply:
We can update those configurations if they pop up